lib/mix/tasks/anonimize_users.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Mix.Tasks.Anonimize.Users do
+  @moduledoc """
+  Task to anonimize users
+  """
+  use Mix.Task
+  import Ecto.Query, warn: false
+
+  alias Safira.Accounts.{Attendee, User}
+  alias Safira.Repo
+
+  def run(_args) do
+    Mix.Task.run("app.start")
+    users = Enum.shuffle(Repo.all(User))
+    n = length(users)
+
+    for i <- 0..(n - 1) do
+      anonimze_user(Enum.at(users, i), i + 1)
+    end
+  end
+
+  defp anonimze_user(%User{} = user, index) do
+    attendee =
+      Attendee
+      |> where([a], a.user_id == ^user.id)
+      |> Repo.one()
+
+    if not is_nil(attendee) do
+      attendee
+      |> Attendee.changeset(%{
+        name: "Attendee #{index}",
+        nickname: "attendee#{index}",
+        avatar: nil,
+        cv: nil
+      })
+      |> Repo.update!()
```

Comment:
Why have you chose to update the database and not just create a file with this information, for example?

Reply:
Because it is easier IMO to manipulate the database rather than a file, because the queries are structured. If it was a file you would need to then process that file, while with a DB that is given to you (pretty much) for free